src/script/components/MessagesList/Message/util/index.ts

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
what about titling this file the name of the hook?

Reply:
I didn't do that on purpose. The functionality is not big enough to be placed into its own file imho. Index is fine in this case and makes the import shorter :)